### PR TITLE
Improve clarity and English.  Hopefully. ;-)

### PR DESCRIPTION
**Pull request by xpmatteo**
I know you just released 1.0.0.  Soooo sorry this is the first time I take a look at it.  Some of the modifications fix what I believe are incorrect or awkward English constructions.  Others are for clarity.

Disclaimer: I have no English guru creds.  I'm just a guy who reads a lot of English.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 [![License: CC BY-SA 4.0](https://img.shields.io/badge/License-CC%20BY--SA%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by-sa/4.0/)
 
-The Frameworkless Movement is a group of developers interested in developing applications without frameworks. We don't hate frameworks, nor we will ever create campaigns against frameworks, but we perceive the **misuse of frameworks as a lack of knowledge regarding technical debt** and the availability of alternatives given by the languages or by dedicated libraries.
+The Frameworkless Movement is a group of developers interested in developing applications without frameworks. We don't hate frameworks, nor we will ever create campaigns against frameworks, but we perceive the **misuse of frameworks as a lack of knowledge regarding technical debt** and we acknowledge the availability of alternatives to frameworks, namely using dedicated libraries, standard libraries, programming languages and operating systems.
```
Comment (xpmatteo) [inline README.md:8]: Here "given ..." we are missing a verb.   "We perceive.... (a) and (b)" but (b) is so far from the verb that it's unclear if "the availablility" is still depending on that verb.

File: README.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ We also want to give the right weight to **technical decision-making** and in re
 
 ## Principles
 
-Everytime that we need to choose a framework for a project we will use these principles to guide our choices.
+Everytime that we consider the choice of a framework for a project we will use these principles to guide our choices.
```
Comment (xpmatteo) [inline README.md:19]: I argue that there is never the "need" of a framework.  There might be the *perception" of the need.

File: README.md [hunk 3]
```diff
@@ -15,15 +15,15 @@ We also want to give the right weight to **technical decision-making** and in re
...
 
 * **The value of a software is not the code itself but in the reasons behind the existence of that code.**
 
-* **Every decision should be made considering the context. A good choice for a context could be a bad choice for another one.**
+* **Every decision should be made considering the context. A good choice in a given context could be a bad choice in another one.**
```
Comment (xpmatteo) [inline README.md:23]: "in" context, not "for a" context

File: README.md [hunk 4]
```diff
@@ -15,15 +15,15 @@ We also want to give the right weight to **technical decision-making** and in re
...
 
-* **The choice of a framework is a technical choice and thus should be made by technical people, but to make mindful decisions business needs should be kept in consideration.**
+* **The choice of a framework is a technical one and should be made by technical people, taking business needs into account.**
```
Comment (xpmatteo) [inline README.md:25]: The original is verbose and hard to read.  You could phrase it in many different ways, but please let's be brief and to the point!

File: README.md [hunk 5]
```diff
@@ -15,15 +15,15 @@ We also want to give the right weight to **technical decision-making** and in re
...
 
-* **The decision-making criteria that made someone to choose a framework should be accessible to all the members in the team.**
+* **The decision-making criteria that led to the choice of a framework should be known to all the members in the team.**
```
Comment (xpmatteo) [inline README.md:27]: You don't "make me to choose" something.  You may "lead me to choose" something.

Also, what do we mean here by "accessilble"?  If we mean "known" let's use known

File: README.md (hunk 1 above)
Comment (francesco-strazzullo) [reply README.md:8]: Yeah I agree with you, it's more clear in this way.

File: README.md (hunk 2 above)
Comment (francesco-strazzullo) [reply README.md:19]: You're right! Removing the "need" is part of our purpose after all :D

**Comment by francesco-strazzullo** [comment]
Thank you very much, your help is really appreciated!

**Pull request merged**